### PR TITLE
Add Item Swing Usage To Food/Potions <=1.7.x

### PR DESCRIPTION
**Pull request by lowercasebtw**
In 1.7.x and below, like with the sword blocking animation, you could use & swing the item on the ground with food/potions also. 
Renamed the field to be appropriate also.

### CODE REVIEW

**Review by FlorianMichael — CHANGES_REQUESTED** [review]
Bow interactions are missing, aren't they?

**Comment by lowercasebtw** [comment]
Oh yea, will push in a second

File: src/main/java/de/florianmichael/viafabricplus/injection/mixin/fixes/minecraft/item/MixinHeldItemRenderer.java [hunk 1]
```diff
@@ -48,15 +48,14 @@ public abstract class MixinHeldItemRenderer {
             slice = @Slice(from = @At(value = "INVOKE", target = "Lnet/minecraft/item/ItemStack;getUseAction()Lnet/minecraft/util/UseAction;")),
             at = @At(value = "INVOKE", target = "Lnet/minecraft/client/render/item/HeldItemRenderer;applyEquipOffset(Lnet/minecraft/client/util/math/MatrixStack;Lnet/minecraft/util/Arm;F)V", ordinal = 2, shift = At.Shift.AFTER))
     private void transformSwordBlockingPosition(AbstractClientPlayerEntity player, float tickDelta, float pitch, Hand hand, float swingProgress, ItemStack item, float equipProgress, MatrixStack matrices, VertexConsumerProvider vertexConsumers, int light, CallbackInfo ci) {
-        final boolean enableItemSwingUsage = VisualSettings.global().enableItemSwingUsage.isEnabled();
-        if (!VisualSettings.global().enableSwordBlocking.isEnabled() && !enableItemSwingUsage) {
```
Comment (FlorianMichael) [inline src/main/java/de/florianmichael/viafabricplus/injection/mixin/fixes/minecraft/item/MixinHeldItemRenderer.java]: This change is breaking, revert.

File: src/main/java/de/florianmichael/viafabricplus/injection/mixin/fixes/minecraft/item/MixinHeldItemRenderer.java [hunk 2]
```diff
@@ -67,6 +67,23 @@ private void transformSwordBlockingPosition(AbstractClientPlayerEntity player, f
         matrices.multiply(RotationAxis.POSITIVE_Z.rotationDegrees(direction * 78.05F));
     }
 
+    @Inject(method = "renderFirstPersonItem",
+            at = @At(value = "INVOKE", target = "Lnet/minecraft/client/render/item/HeldItemRenderer;applyEquipOffset(Lnet/minecraft/client/util/math/MatrixStack;Lnet/minecraft/util/Arm;F)V", ordinal = 3, shift = At.Shift.AFTER))
+    private void applyFoodSwingOffset(AbstractClientPlayerEntity player, float tickDelta, float pitch, Hand hand, float swingProgress, ItemStack item, float equipProgress, MatrixStack matrices, VertexConsumerProvider vertexConsumers, int light, CallbackInfo ci) {
+        if (VisualSettings.global().enableItemSwingUsage.isEnabled()) {
```
Comment (FlorianMichael) [inline src/main/java/de/florianmichael/viafabricplus/injection/mixin/fixes/minecraft/item/MixinHeldItemRenderer.java:73]: Maybe this code should be moved to a separate method

**Review by FlorianMichael — CHANGES_REQUESTED** [review]
we should rename the setting to "Swing hand on item use" or something like that.

**Review by FlorianMichael — APPROVED** [review]

**Pull request merged**